test/helpers.bash

```diff
@@ -499,3 +499,20 @@ function cleanup_network_conf() {
 function temp_sandbox_conf() {
 	sed -e s/\"namespace\":.*/\"namespace\":\ \"$1\",/g "$TESTDATA"/sandbox_config.json > $TESTDIR/sandbox_config_$1.json
 }
+
+function wait_until_exit() {
+	ctr_id=$1
+	# Wait for container to exit
+	attempt=0
+	while [ $attempt -le 10 ]; do
+		attempt=$((attempt+1))
+		run crictl inspect "$ctr_id" --output table
+		echo "$output"
+		[ "$status" -eq 0 ]
+		if [[ "$output" =~ "State: CONTAINER_EXITED" ]]; then
+			[[ "$output" =~ "Exit Code: 0" ]]
+			break
+		fi
+		sleep 1
+	done
```

Comment:
I think we want to `return 1` here if we fall out of the loop (and then check for returned errors when calling `wait_until_exit`).

Reply:
added a `return 1` if we hit the whole timeout.